springbootsample/build.gradle

```diff
@@ -0,0 +1,28 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.springframework.boot:spring-boot-gradle-plugin:1.5.19.RELEASE'
+    }
+}
+
+apply plugin: 'org.springframework.boot'
+apply plugin: 'java'
+apply plugin: 'io.spring.dependency-management'
+
+group = 'com.microsoft.applicationinsights'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '1.8'
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    compile 'org.springframework.boot:spring-boot-starter-web'
+    compile 'org.apache.httpcomponents:httpclient:4.5.7'
+    compile project(':azure-application-insights-spring-boot-starter')
```

Comment:
when a customer would like to create a sample app in independent project he can just have the dependency like
`compile 'com.microsoft.azure:applicationinsights-spring-boot-starter:1.1.2'` instead. 
The current project import is just because we are having this sample inside the project repository.